app/controllers/api/v1x0/mixins/index_mixin.rb

```diff
@@ -15,8 +15,8 @@ def collection(base_query)
           resp = ManageIQ::API::Common::PaginatedResponse.new(
             :base_query => filtered(scoped(base_query)),
             :request    => request,
-            :limit      => pagination_limit,
-            :offset     => pagination_offset
+            :limit      => params[:limit],
```

Comment:
Make no change to the mixin which applies to all classes. Override the `collection` method only in the `Workflow` class.